src/services/groups.service.ts

```diff
@@ -0,0 +1,96 @@
+import { PrismaClient, Group } from '@prisma/client';
+import { CreateGroupDto } from '@dtos/group.dto';
+import { HttpException } from '@exceptions/HttpException';
+import { isEmpty } from '@utils/util';
+
+class GroupService {
+  public group = new PrismaClient().group;
+
+  public async findAllGroup(): Promise<Group[]> {
+    const allGroup: Group[] = await this.group.findMany();
+    return allGroup;
+  }
+
+  public async findGroupById(groupId: number): Promise<Group> {
+    if (isEmpty(groupId)) throw new HttpException(400, "groupId is empty");
+
+    const findGroup = await this.group.findUnique({ where: { id: groupId }});
+    if (!findGroup) throw new HttpException(409, "Group doesn't exist");
+
+    return findGroup;
+  }
```

Comment:
Change the name of the function. As we are on the **Groups Service**, if we have a method that returns the **groups** we can simply call it like findById because it tells that we are listing the entity from which our service is responsible.

I really like the idea of always use braces, even that its just one line, and also break the line.
Adding braces, it makes more easier to maintain it and to not add new code into a different scope.

Also, adding braces and break lines makes the code a lot easier to read.

On line 15 and 18 try to avoid Magic Numbers.
On line 17 the variable name could be **groupFound**